modules/core_api/fsw/inc/cfe_evs_extern_typedefs.h

```diff
@@ -0,0 +1,179 @@
+/*
+**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**  Copyright (c) 2006-2019 United States Government as represented by
+**  the Administrator of the National Aeronautics and Space Administration.
+**  All Rights Reserved.
+**
+**  Licensed under the Apache License, Version 2.0 (the "License");
+**  you may not use this file except in compliance with the License.
+**  You may obtain a copy of the License at
+**
+**    http://www.apache.org/licenses/LICENSE-2.0
+**
+**  Unless required by applicable law or agreed to in writing, software
+**  distributed under the License is distributed on an "AS IS" BASIS,
+**  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**  See the License for the specific language governing permissions and
+**  limitations under the License.
+*/
+
+/**
+ * @file
+ *
+ * Declarations and prototypes for cfe_evs_extern_typedefs module
+ */
+
+#ifndef CFE_EVS_EXTERN_TYPEDEFS_H
+#define CFE_EVS_EXTERN_TYPEDEFS_H
+
+/* This header may be generated from an EDS file,
+ * tools are available and the feature is enabled */
+#ifdef CFE_EDS_ENABLED_BUILD
+
+/* Use the EDS generated version of these types */
+#include "cfe_evs_eds_typedefs.h"
+
+#else
+/* Use the local definitions of these types */
+
+#include "common_types.h"
+
+/**
+ * @brief Label definitions associated with CFE_EVS_MsgFormat_Enum_t
+ */
+enum CFE_EVS_MsgFormat
+{
+
+    /**
+     * @brief Short Format Messages
+     */
+    CFE_EVS_MsgFormat_SHORT = 0,
+
+    /**
+     * @brief Long Format Messages
+     */
+    CFE_EVS_MsgFormat_LONG = 1
+};
+
+/**
+ * @brief Identifies format of log messages
+ *
+ * @sa enum CFE_EVS_MsgFormat
+ */
+typedef uint8 CFE_EVS_MsgFormat_Enum_t;
```

Comment:
if the intent is to only have SHORT or LONG MsgFormat, why not use a bool? this would be 1byte vs an enum which is most likely 4 bytes, just to but it into a uint8 which is 1 byte

Reply:
This could be extended and we avoid use of bool in cmd/tlm.